.azure/pipelines/jobs/fuzzing.yml

```diff
@@ -0,0 +1,29 @@
+parameters:
+  method: ''
+
+jobs:
+  - job: ${{ parameters.method }}
+    pool:
+      vmImage: 'ubuntu-18.04'
+    timeoutInMinutes: 0     # 360 minutes
+    steps:
+      - task: UseDotNet@2
+        displayName: 'Use dotnet sdk 3.x'
+        inputs:
+          version: $(SDK_VERSION)
+          includePreviewVersions: true
+      
+      - bash: |
+          cd fuzz
+          chmod +x *.sh
+
+          sudo ./setup.sh
+          export PATH="$PATH:/root/.dotnet/tools"
+
+          ./init.sh
+          ./run.sh $(TIMEOUT) ${{ parameters.method }}
+
+      - task: PublishBuildArtifacts@1
```

Comment:
Should run only if _failed_, otherwise there are no files in the _crashes_-folder.